editors/vscode/src/common.ts

```diff
@@ -293,9 +293,21 @@ async function maybeSendStartupTelemetryEvent(
     telemetryLogger.logUsage("extension-activated", usageData);
 }
 
-const HELP_URL = "https://snapshots.slint.dev/master/docs/slint/reference/";
+function helpBaseUrl(context: vscode.ExtensionContext): string {
+    if (
+        context.extensionMode === vscode.ExtensionMode.Development ||
+        context.extension.packageJSON.name.endsWith("-nightly")
+    ) {
+        return "https://snapshots.slint.dev/master/docs/slint/reference/";
+    } else {
+        return `https://docs.slint.dev/${context.extension.packageJSON.version}/docs/slint/reference/`;
```

Comment:
```suggestion
        return `https://releases.slint.dev/${context.extension.packageJSON.version}/docs/slint/reference/`;
```

(I think)

Reply:
I think we should be consistent to what we use in other places And we use releases.slint.dev everywhere with a version.
See for example https://github.com/slint-ui/slint/blob/5e778714de8d24140945e7ff4e7e3fffb795aeec/internal/core-macros/slint_doc.rs#L50

So I think we should use that.

Reply:
Good point :)